src/containers/api.js

```diff
@@ -0,0 +1,16 @@
+const clothingCategory=['top','bottom']
+
+const stylesByCategory = {
+  top: ['short-sleeve', 'long-sleeve', 'sleeveless'],
+  bottom: ['skirt', 'pants', 'shorts'],
+};
+
+const clothingColor={
+  colors:['black','white','blue','red','other']
+}
```

Comment:
I don't think you need the object data structure for `clothingColor` and `clothingSeason`. A plain array should be enough, just like you've done for `clothingCategory`. I also suggest you make the variable names plural for consistency:
```
const clothingCategories = ['top', 'bottom'];
const clothingColors = ['black', 'white', 'blue', 'red', 'other'];
const clothingSeasons = ['fall/winter', 'summer/spring'];
```